pluma/dialogs/Makefile.am

```diff
@@ -1,12 +1,13 @@
 uidir = $(datadir)/pluma/ui/
 
-AM_CPPFLAGS = 							\
-	-I$(top_srcdir)						\
-	-I$(top_builddir)					\
-	-I$(top_srcdir)/pluma 					\
-	-I$(top_builddir)/pluma					\
-	$(PLUMA_CFLAGS) 					\
-	$(WARN_CFLAGS)
+AM_CPPFLAGS = 					\
+	-I$(top_srcdir)				\
+	-I$(top_builddir)			\
+	-I$(top_srcdir)/pluma 			\
+	-I$(top_builddir)/pluma			\
+	$(PLUMA_CFLAGS) 			\
+	$(WARN_CFLAGS)				\
+	-DPLUMA_DATADIR=\""$(datadir)/pluma"\"
```

Comment:
Why do you need to define `PLUMA_DATADIR` again? It is already defined in `pluma/Makefile.am`

Reply:
Because `PLUMA_DATADIR` is used in the files below:
```
pluma/dialogs/pluma-encodings-dialog.c: ret = pluma_utils_get_ui_objects (PLUMA_DATADIR "/ui/pluma-encodings-dialog.ui",
pluma/dialogs/pluma-preferences-dialog.c:       ret = pluma_utils_get_ui_objects (PLUMA_DATADIR "/ui/pluma-preferences-dialog.ui",
pluma/dialogs/pluma-search-dialog.c:    ret = pluma_utils_get_ui_objects (PLUMA_DATADIR "/ui/pluma-search-dialog.ui",
```

Reply:
@mbkma Do you want to move `PLUMA_DATADIR` into `config.h`?

Reply:
No, I think it is fine.

Without  `-DPLUMA_DATADIR=\""$(datadir)/pluma"\"`   I now get errors as expected `error: ‘PLUMA_DATADIR’ undeclared`. I did not see those errors at a first try, had to run `make clean && make` to see them.